### PR TITLE
Add network VLAN Spanning support

### PR DESCRIPTION
**Pull request by stevestrutt**
Added resource and unit tests for VLAN Spanning configuration on IaaS.

### CODE REVIEW

File: ibm/resource_ibm_network_vlan_spanning.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+func resourceIBMNetworkVlanSpan() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMNetworkVlanSpanCreate,
+		Read:     resourceIBMNetworkVlanSpanRead,
+		Update:   resourceIBMNetworkVlanSpanUpdate,
+		Delete:   resourceIBMNetworkVlanSpanDelete,
+		Importer: &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
```
[the remark this answers is not in the document]
Comment (stevestrutt) [reply ibm/resource_ibm_network_vlan_spanning.go]: Provider validation fails with an error that Delete is required. Left as is.

File: website/docs/r/network_vlan_spanning.html.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+The following arguments are supported:
+
+* `vlan_spanning` - (Optional, string) The desired state of VLAN spanning for the account. Accepted values are `on`, `off`.
+
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `vlan_spanning` - The current state of the IaaS VLAN Spanning attribute for the account. Accepted values are `on`, `off`.
```
[the remark this answers is not in the document]
Comment (stevestrutt) [reply website/docs/r/network_vlan_spanning.html.markdown]: Changed

File: ibm/resource_ibm_network_vlan_spanning.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"vlan_spanning": {
+				Type:         schema.TypeString,
+				Optional:     true,
```
[the remark this answers is not in the document]
Comment (stevestrutt) [reply ibm/resource_ibm_network_vlan_spanning.go]: Changed to required

File: ibm/resource_ibm_network_vlan_spanning.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (hkantare) [reply ibm/resource_ibm_network_vlan_spanning.go]: Not Delete ...remove the `Timeouts: &schema.ResourceTimeout{` section

**Pull request merged**